projects/zlux-file-explorer/src/lib/components/tree/tree.component.ts

```diff
@@ -49,26 +49,31 @@ export class TreeComponent implements AfterContentInit, OnDestroy {
   @Output() rightClickEvent = new EventEmitter<MouseEvent>();
   @Output() panelRightClickEvent = new EventEmitter<MouseEvent>();
   selectedNode: FileNode;
-  lastClickedNodeName: string; // PrimeNG as of 6.0 has no native double click support for its tree
-  lastClickedNodeTimeout: number = 500; // < 500 ms becomes a double click
+  doubleClickTimer : number = 300; // Interval of 300ms or less between two clicks is considered a double-click here.
```

Comment:
Small correction,`doubleClickTimer` isn't a timer right? More like a CONST or a timeout value?

Reply:
Thanks Lenny. I addressed it. Also, I would prefer to keep this variable inside the class, since its only going to be used there. And since class members can't be declared using the const keyword, I'll mark it as private readonly instead to provide an extra layer of protection